nb_conda_kernels/nb_conda_kernels.py

```diff
@@ -116,6 +101,9 @@ def find_kernel_specs(self):
         elif "R" in kspecs:
             kspecs.pop("R")
 
+        # update conda info
+        self.conda_info = self._conda_info()
+
```

Comment:
On lines 118-119, it would be great it we could avoid calling conda_kspecs() twice